src/instances/Instances.js

```diff
@@ -263,58 +263,63 @@ class InstanceManager extends EventEmitter {
 
             // Resolve executable
             try {
-                const releaseNode = this.releaseManager.getRelease(instanceNode.versionrange, instanceNode.channel, true);
-                if (!releaseNode) {
+                this.releaseManager.getOrInstallRelease(instanceNode.versionrange, instanceNode.channel).then((releaseNode) => {
+                    this.log.debug(`Starting instance using release ${releaseNode.version} @ ${releaseNode.channel}`);
+                    this.log.debug('1');
+                    const executable_folder = this.releaseManager.getBinaryPath(releaseNode.version, releaseNode.channel);
+                    this.log.debug('2');
```

Comment:
maybe remove these 1-2-3 messages again?

Reply:
lol, sure :-)

Reply:
The good old log debugging 💓